site/docs/reference/Connectors/capture-connectors/MariaDB/MariaDB.md

```diff
@@ -195,7 +195,7 @@ captures:
 
 Your capture definition will likely be more complex, with additional bindings for each table in the source database.
 
-[Learn more about capture definitions.](/concepts/captures.md#pull-captures)
+[Learn more about capture definitions.](../../../../concepts/captures.md)
```

Comment:
Is it not preferable to use root relative paths rather than document relative paths?

Reply:
hmm yeah you're right, updated these changes. still need to remove the non-existant anchor